native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.ios.js.snap

```diff
@@ -90,6 +90,7 @@ exports[`iOS BpkButton should render correctly 1`] = `
               "lineHeight": 18,
             },
             Object {
+              "fontFamily": "System",
```

Comment:
This seems extraneous, similar to https://github.com/Skyscanner/backpack/pull/177#discussion_r143409735 but a little bit more prevalent - in iOS `"fontFamily": "System",` is already being applied - seems like we can omit the `fontFamily` prop from the emphasize styles if it's iOS?